site2/docs/io-tcp.md

```diff
@@ -0,0 +1,18 @@
+---
+id: io-tcp
+title: Netty Tcp Connector
+sidebar_label: Netty Tcp Connector
+---
+
+## Source
+
+The Netty Tcp Source connector is used to listen Tcp messages from
```

Comment:
I would suggest adding a comment - this connector is suggested to be used in a containerized (e.g. k8s) deployment. otherwise, if the connector is running in process or thread mode, the instances will be conflicting on listening to ports.